### ISSUE TITLE
Font color / size for a portion of a text (span)?

### ISSUE THREAD

**Issue by borekb** [jxnblk/mdx-deck#249]
Hi, I'm new to mdx-deck and wanted to have one specific line in my deck in different color and perhaps a larger font as well. It was not very obvious to me how to achieve that. I tried the following:

- `<span style="color: red">My text</span>` but it crashes – it's valid Markdown but not valid JSX (probably the cause?)
- Using `<span style={{ color: 'red' }}>My text</span>` which works but makes the font smaller. I don't quite understand why.
- Also, the above crashes if there's more text outside the tag, like `Hello <span style={{ color: 'red' }}>world</span>`.
- `Hello <font color="red">world</font>` seems to render correctly. 🎉 
- `<font color="red">Hello world</font>` has that font size issue again.

Maybe another approach would be to define a class / className and style it with CSS? I'm not entirely sure, some hints or docs would be appreciated. Thanks!

**Comment by jxnblk** [comment]
You might want to try using `<p>` instead of `<span>` – or at least wrapping the `<span>` in a `<p>` block so that MDX parses the entire block as JSX. And, although I love the `<font>` tag, MDN [recommends avoiding it](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font):

> This feature is obsolete. Although it may still work in some browsers, its use is discouraged since it could be removed at any time. Try to avoid using it.

**Comment by borekb** [comment]
Yeah, `<font>` was something I didn't want to do :) And it's not a perfect workaround, for example, I'd like the "toned down" text to adapt to a theme, so hardcoding a specific color didn't feel right.

BTW, during my initial experience with mdx-deck, it was not entirely clear to me what is the boundary between Markdown and JSX. I should read up more on MDX, that's probably the base layer which I should understand. Sorry if this issue doesn't belong here.

**Comment by borekb** [comment]
I ended up creating a custom `.muted` class:

![screenshot 2019-02-06 at 11 35 16](https://user-images.githubusercontent.com/101152/52335721-5de3f600-2a03-11e9-8b47-b82963476b8d.png)

My `theme.js` looks like this:

```js
import theme from "mdx-deck/themes";

export default {
  ...theme,
  css: {
    ...theme.css,
    ".muted": {
      color: "#808080"
    }
  }
};
```

Together with the fix for nested list items (#198), it looks like this:

```js
import theme from "mdx-deck/themes";

export default {
  ...theme,
  css: {
    ...theme.css,
    "li ul, li ol": {
      fontSize: "1em"
    },
    "li > p": {
      fontSize: "1em",
      margin: 0
    },
    ".muted": {
      color: "#808080"
    }
  }
};

```

**Comment by jxnblk** [comment]
That seems like a fine solution. Feel free to reopen this if you had further questions
